src/regex_split.cpp

```diff
@@ -122,10 +122,28 @@ void RegexSplit::validate_and_infer_types() {
 }
 
 bool RegexSplit::evaluate(ov::TensorVector& outputs, const ov::TensorVector& inputs) const {
+    const size_t num_chars = inputs[4].get_size();
     auto input_size = get_input_size();
     const bool has_skips = (input_size == 7);
 
+    if (num_chars == 0) {
+        // TODO: CVS-160624. This is a workaround for the case when the input is empty.
+        outputs[0].set_shape(Shape{1});
+        outputs[1].set_shape(Shape{1});
+        outputs[0].data<int32_t>()[0] = 0;
+        outputs[1].data<int32_t>()[0] = 0;
+
+        outputs[2] = inputs[2];
+        outputs[3] = inputs[3];
+        outputs[4] = inputs[4];
+        if (has_skips) {
+            outputs[5] = inputs[5];
+        }
+        return true;
```

Comment:
let us cover this case in separate PR. We need a test for this case.

Reply:
please revert change

Reply:
Without this w/a we will have problems with empty strings with PR and PR will become useless. On the other hand addressing this issue will require extra efforts in other operations which are not quite related to this PR.

We currently already have this bug with the current master, but in other operations we also have the counter-bug which neutralizes this bug: either we change all operations or we fallback to old (even though it was not correct) behavior to get the same output. We opted with @apaniukov for the moment the second choice and planed to revise separately.